src/Quizzes/QuizEditor/QuestionsEditor/QuizQuestionsEditor.tsx

```diff
@@ -216,19 +217,25 @@ const QuizQuestionsEditor = () => {
 
     } else if (type === 'fill-blanks') {
       setCorrectAnswers(["Example 1", "Example 2", "Example 3", "Example 4"]);
+      setEditingQuestion({ ...(editingQuestion as FillInBlank), correctAnswers: ["Example 1", "Example 2", "Example 3", "Example 4"] });
     }
     setQuestionType(e.target.value);
   }
 
-  const handlePublishQuiz = () => {
-    // Implement logic to publish the quiz
+  const handlePublishQuiz = async () => {
+    handleSaveQuiz(); 
+    const response = await api.put(`${QUIZZES_API}/quizzes/${quizId}/updateQuizPublish`);
   };
 
   const handleSaveQuiz = async () => {
-    // we have to package questions into a json
     const jsonQuestions = questionsToJson(questions);
-    const response = await api.post(`${QUIZZES_API}/quizzes/${quizId}/questions`, jsonQuestions);
-    console.log(response);
+    console.log(`${QUIZZES_API}/quizzes/${quizId}/questions`);
+    const response1 = await api.post(`${QUIZZES_API}/quizzes/${quizId}/questions`, jsonQuestions);
+
+    const jsonDetails = {points: totalPoints, numQuestions: questions.length};
+    const response2 = await api.put(`${QUIZZES_API}/quizzes/${quizId}/updatePointsAndNumQuestions`, jsonDetails);
```

Comment:
we've lowkey created inconsistency with our routes due to courses/quizzes/ vs /quizzes/ but honestly if this works i'm good with it
if this were production or something I'd be way more incensed ig lol